airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml

```diff
@@ -297,7 +297,46 @@ definitions:
       data_field: "conversations"
       page_size: 150
 
+  # activity logs stream is incremental based on created_at field
+  activity_logs:
+    $ref: "#/definitions/stream_full_refresh"
+    primary_key: id
+    $parameters:
+      name: "activity_logs"
+      path: "admins/activity_logs"
+      data_field: "activity_logs"
+    retriever:
+      $ref: "#/definitions/retriever"
+      description: "The Retriever without passing page size option"
+      paginator:
+        type: "DefaultPaginator"
+        url_base: "#/definitions/requester/url_base"
+        pagination_strategy:
+          type: "CursorPagination"
+          cursor_value: "{{ response.get('pages', {}).get('next') }}"
+          stop_condition: "{{ 'next' not in response.get('pages', {}) }}"
+        page_token_option:
+          type: RequestPath
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: created_at
+      cursor_datetime_formats:
+        - "%s"
+      datetime_format: "%s"
+      cursor_granularity: "PT0.000001S"
```

Comment:
created_at property is formatted as a UNIX timestamp (in seconds).
Is there any reason to set `cursor_granularity` in microseconds?

Reply:
Fixed